moonshot/src/api/api_bookmark.py

```diff
@@ -0,0 +1,104 @@
+from moonshot.src.bookmark.bookmark import Bookmark
+from moonshot.src.bookmark.bookmark_arguments import BookmarkArguments
+
+_bookmark_instance = None
+
+
+def get_bookmark_instance():
+    global _bookmark_instance
```

Comment:
@imda-normanchia would prefer not to have a global var here.
is it possible to read and just return the value instead since Bookmark is already a singleton pattern